Assets/Scripts/Combat/HealthBar.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthBar : MonoBehaviour
```

Comment:
Is this a script that's also used by the shield bar? If so it could be renamed to something like `PointsBar`.

Reply:
Hm I thought healthbar was general enough but sure I'll change it to PointsBar